src/votekit/ballot_generator.py

```diff
@@ -9,9 +9,9 @@
 import random
 from typing import Optional
 
-from .ballot import Ballot
+from .ballot import Ballot, PointBallot
 from .pref_profile import PreferenceProfile
-
+import apportionment.methods as apportion # type: ignore
```

Comment:
to ensure this works as a package, this should be a relative import like the ones above (start with one or more `.` to define the relative path between them

Reply:
nevermind, thought this was an internal module